lntest/btcd.go

```diff
@@ -81,6 +81,7 @@ func NewBackend(miner string, netParams *chaincfg.Params) (
 		"--debuglevel=debug",
 		"--logdir=" + logDir,
 		"--connect=" + miner,
+		"--nowinservice",
```

Comment:
itest now in service 👮

Reply:
Haha, cannot be unseen. Didn't think of that :rofl: